diff --git a/ios/Toolbar/RouterToolbarModule.swift b/ios/Toolbar/RouterToolbarModule.swift
index 1bee03ce614c6169c56ff74cdf89128923c500f5..e41e4621039767f3d0a0a1e3266ebca625507228 100644
--- a/ios/Toolbar/RouterToolbarModule.swift
+++ b/ios/Toolbar/RouterToolbarModule.swift
@@ -71,10 +71,10 @@ public class RouterToolbarModule: Module {
         view.titleStyle = style?.toTitleStyle()
       }
       Prop("accessibilityLabel") { (view: RouterToolbarItemView, accessibilityLabel: String?) in
-        view.accessibilityLabel = accessibilityLabel
+        view.routerAccessibilityLabel = accessibilityLabel
       }
       Prop("accessibilityHint") { (view: RouterToolbarItemView, accessibilityHint: String?) in
-        view.accessibilityHint = accessibilityHint
+        view.routerAccessibilityHint = accessibilityHint
       }
       Prop("disabled") { (view: RouterToolbarItemView, disabled: Bool?) in
         view.disabled = disabled ?? false
